pytorch_lightning/core/lightning.py

```diff
@@ -1948,11 +1948,14 @@ def add_to_queue(self, queue: torch.multiprocessing.SimpleQueue) -> None:
 
         Args:
             queue: the instance of the queue to append the data.
+
+        .. deprecated:: v1.5
+            This method was deprecated in v1.5 in favor of
+            `DDPSpawnPlugin.add_to_queue`
+            and will be removed in v1.7.
         """
-        callback_metrics: dict = apply_to_collection(
-            self.trainer.callback_metrics, torch.Tensor, lambda x: x.cpu().numpy()
-        )  # send as numpy to avoid issues with memory sharing
-        queue.put(callback_metrics)
+        if self.trainer and isinstance(self.trainer.training_type_plugin, pl.plugins.training_type.DDPSpawnPlugin):
```

Comment:
there shall be also raised warning

Reply:
this is done in configuration_validator to avoid sending multiple warnings